doc/api/esm.md

```diff
@@ -1139,11 +1139,18 @@ CommonJS modules loaded.
 
 ### Hooks
 
-#### <code>resolve</code> hook
+#### <code>resolve(specifier, parentURL, context, defaultResolve)</code>
```

Comment:
Doesn't have to be addressed in this PR, but do we know why we're using `<code>` here rather than ``` ` ```?

Reply:
Sorry, I do not know the answer to that. Perhaps @GeoffreyBooth would know?

Reply:
It's possible `<code>` was being used when ``` ` ``` didn't work in headers, which was until relatively recently.

Reply:
Anyway, totally not blocking or anything. Just curious.

Reply:
> Doesn't have to be addressed in this PR, but do we know why we're using `<code>` here rather than <code>`</code>?

I though I remembered Geoffrey mentioning something about this, but may only slightly related.

> I think if we put the signature in the header, then the permalink will change every time the signature changes. Could we perhaps leave this as it was?
&mdash; https://github.com/nodejs/node/pull/33812#discussion_r439097360

I speculate that `<code>` may have been continued to be used for a couple of reasons:
&emsp;a) we weren't aware that backticks in headings were now supported
&emsp;b) we didn't want to change the permalink, which includes `code` in the fragment if the HTML tag is used

I've gone ahead and fixed it since:
&emsp;a) the permalink will be changing
&emsp;b) having `code` in the fragment undesirable

Good catch!